spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/actuate/GatewayControllerEndpoint.java

```diff
@@ -68,10 +66,9 @@
 
 	private ApplicationEventPublisher publisher;
 
-	public GatewayControllerEndpoint(RouteDefinitionLocator routeDefinitionLocator,
-			List<GlobalFilter> globalFilters, List<GatewayFilterFactory> GatewayFilters,
+	public GatewayControllerEndpoint(List<GlobalFilter> globalFilters,
+			List<GatewayFilterFactory> GatewayFilters,
```

Comment:
Why isn't the checkstyle flagging `GatewayFilters` as a naming convention error? 🤔

Reply:
@marcingrzejszczak ?

Reply:
fixed in de8bcdfe64c856911c1aa3b85bd436b661b539ae